benchmark/src/ruben_opt.rs

```diff
@@ -72,6 +72,10 @@ pub struct Opt {
     /// If not specified or equals 0, it will be set to validator_addresses.len().
     #[structopt(short = "c", long = "num_clients", default_value = "0")]
     pub num_clients: usize,
+    /// Exclusive upper bound value in milliseconds to stagger the client
+    /// for some randomly time before submitting TXNs. Minimum 1 ms since it is exclusive.
```

Comment:
I'd change the comment to describe the desired action: Randomly distribute the clients to start sending requests over the stagger_range_ms time. A value of 1 means effectively means starting all clients at once